code/modules/virus2/effect.dm

```diff
@@ -861,7 +861,12 @@ var/list/compatible_mobs = list(/mob/living/carbon/human, /mob/living/carbon/mon
 	name = "Itching"
 	stage = 1
 /datum/disease2/effect/itching/activate(var/mob/living/carbon/mob,var/multiplier)
-	to_chat(mob, "<span class='warning'>Your skin itches!</span>")
+	var/mob/living/carbon/human/H = mob
+	if (istype(H) && H.species.flags & NO_SKIN)
```

Comment:
Or, if you want to be SUPER extra careful, you could replace the second check there with `istype(H.species)`